### PR TITLE
All Neighbors System Dashboard - Phase 1 (for launch)

### PR DESCRIPTION
**Pull request by eanders**
[//]: # 'remove this if PR is for a release-* branch'
# _Please squash merge this PR_

## Description

Updates for ANSD

## Type of change
[//]: # 'remove options that are not relevant'
- [x] New feature (adds functionality)

## Checklist before requesting review
- [x] I have performed a self-review of my code
- [x] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [x] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [x] My code follows the style guidelines of this project (rubocop)
- [x] I have updated the documentation (or not applicable)
- [x] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

File: app/assets/javascripts/warehouse_reports/all_neighbors_system_dashboard/stack.js [hunk 1]
```diff
@@ -128,14 +128,16 @@ class AllNeighborsSystemDashboardStack {
         contents: (d, defaultTitleFormat, defaultValueFormat, color) => {
           const index = d[0].index
           const title = this.series[index].name
+          console.log(d, title)
```
Comment (awisema) [inline app/assets/javascripts/warehouse_reports/all_neighbors_system_dashboard/stack.js]:
```suggestion
          console.debug(d, title)
```
?

File: app/assets/javascripts/warehouse_reports/all_neighbors_system_dashboard/stack.js [hunk 2]
```diff
@@ -329,6 +331,7 @@ class AllNeighborsSystemDashboardTTOHStack extends AllNeighborsSystemDashboardSt
       return col.concat(this.series.map((d) => d.name))
     } else {
       const index = this.config.keys.indexOf(name)
+      let household_count = 0
```
Comment (awisema) [inline app/assets/javascripts/warehouse_reports/all_neighbors_system_dashboard/stack.js]: A minor quibble, but, I think we should follow JS coding conventions in JS, so this should be `householdCount`?

File: app/assets/javascripts/warehouse_reports/all_neighbors_system_dashboard/stack.js [hunk 3]
```diff
@@ -400,10 +409,9 @@ class AllNeighborsSystemDashboardTTOHStack extends AllNeighborsSystemDashboardSt
       tooltip: {
         contents: (d, defaultTitleFormat, defaultValueFormat, color) => {
           const index = d[0].index
-          const householdTotal = d3.sum(
-            this.series[index].series.filter((n) => this.inDateRange(n.date, this.state.dateRange)),
-            (n) => n.household_count  
-          )
+          let household_counts = this.series[index].series.filter((n) => this.inDateRange(n.date, this.state.dateRange))
```
Comment (awisema) [inline app/assets/javascripts/warehouse_reports/all_neighbors_system_dashboard/stack.js]: `householdCounts`?

File: docker/docker-compose.yml [hunk 4]
```diff
@@ -121,6 +121,7 @@ services:
     expose:
       - "3000"
       - "9394"
+      - "12345"
```
Comment (awisema) [inline docker/docker-compose.yml]: Not sure why this is here?

File: drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/enrollment.rb [hunk 5]
```diff
@@ -26,14 +26,19 @@ class Enrollment < GrdaWarehouseBase
     # To be considered housed (or "placed") one of the following conditions must be met
     # 1. Client had an exit within the date range and the Exit was to a permanent destination and from a diversion project (specified at run-time)
     # 2. Client has a move-in date during range
-    scope :housed_in_range, ->(range, filter:) do
-      permanent_diversion_exit(range, filter: filter).
-        or(moved_in_in_range(range, filter: filter))
+    # NOTE: replaced with `placed_in_range` which uses the normalized placment date
```
Comment (awisema) [inline drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/enrollment.rb]:
```suggestion
    # NOTE: replaced with `placed_in_range` which uses the normalized placement date
```

File: drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/housing_total_placements_data.rb [hunk 6]
```diff
@@ -1,7 +1,14 @@
...
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
 module AllNeighborsSystemDashboard
   class HousingTotalPlacementsData < DashboardData
     include AllNeighborsSystemDashboard::CensusCalculations
 
+    # FIXME: need to figure out why we are no longer getting the "All" line
```
Comment (awisema) [inline drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/housing_total_placements_data.rb]: Is this still an issue?

File: drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/housing_total_placements_data.rb [hunk 7]
```diff
@@ -112,6 +129,7 @@ def data(title, id, type, options: {})
     end
 
     def line(options, fixed_start_date: nil, count_item:)
+      # binding.pry if options[:project_type] == 'Diversion'
```
Comment (awisema) [inline drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/housing_total_placements_data.rb]:
```suggestion
```

File: drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/housing_total_placements_data.rb [hunk 8]
```diff
@@ -166,7 +183,7 @@ def line_data
     def donut(options, count_item:, **)
       project_type = options[:project_type] || options[:homelessness_status]
       options[:types].map do |type|
-        # binding.pry #if type == 'project_type'
+        # binding.pry #if project_type == 'Diversion'
         # raise 'failed'
```
Comment (awisema) [inline drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/housing_total_placements_data.rb]:
```suggestion
```

File: drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/housing_total_placements_data.rb [hunk 9]
```diff
@@ -189,6 +206,7 @@ def donut(options, count_item:, **)
 
     def donut_data
       [
+        # FIXME: this isn't calculating correctly, it looks like we're distincting somewhere
```
Comment (awisema) [inline drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/housing_total_placements_data.rb]: Is this still an issue?

**Review by awisema — CHANGES_REQUESTED** [review]

File: docker/docker-compose.yml (hunk 4 above)
Comment (eanders) [reply docker/docker-compose.yml]: Oh, thanks, that was an attempt a remote debugging.  Guess I missed it.

**Review by awisema — APPROVED** [review]

**Pull request merged**